app/controllers/v1/collection_items_controller.rb

```diff
@@ -0,0 +1,25 @@
+class V1::CollectionItemsController < ApplicationController
+  before_action :pundit_user
+
+  def index
+    @user = User.find_by(username: params[:user_username])
+    @collection = @user.collection
+    render :index, status: :ok
+  end
+
+  def create
+    @item = @current_user.items.build(item_params)
```

Comment:
No authorization needed?

Reply:
Nope... User only needs to be authenticated to add a "book" in his/her collection. Book objects are the ones that need admin access.